config/development.json

```diff
@@ -64,7 +64,7 @@
 		"external-media/openverse": true,
 		"fediverse/allow-opt-in": true,
 		"full-site-editing/beta-opt-in": true,
-		"cookie-banner": false,
+		"cookie-banner": true,
```

Comment:
You can also use `?flags=cookie-banner` for testing. Or change these, but they shouldn't be committed to prod. :)

Reply:
Thanks @gmovr ! I have changed back the option.
`?flags=cookie-banner` is a great tip! 
The setup has been a little confusing for me as I am not currently in a GDPR country and had to try different things to get the banner to show up 😅

Reply:
> I am not currently in a GDPR country and had to try different things to get the banner to show up

The whole cookie banner depends on the `country_code` cookie - so you can easily switch countries by modifying it. You can use dev tools to change it to some European country e.g. `NL`.